.github/workflows/node.js.yml

```diff
@@ -15,62 +15,20 @@ jobs:
       with:
         access_token: ${{ github.token }}
 
-  install:
+  build:
     needs: check-skip
     runs-on: ubuntu-latest
     steps:
     - uses: actions/checkout@v2
     - uses: actions/setup-node@v2
       with: 
-        node-version: 14
-    - name: node_modules cache
-      id: cache
-      uses: actions/cache@v2
-      with:
-        path: |
-          **/node_modules
-        key: ${{ github.run_number }}
+        node-version: 16
+        registry-url: 'https://registry.npmjs.org'
     - name: Install dependencies
       run: |
-        npx -p @s-ui/mono@1 sui-mono phoenix --ci
-        npm run install:demos -- --ci
-        
-  lint:
-    needs: install
-    runs-on: ubuntu-latest
-    steps:
-    - uses: actions/checkout@v2
-    - uses: actions/setup-node@v2
-      with: 
-        node-version: 14
-    - name: node_modules cache
-      id: cache
-      uses: actions/cache@v2
-      with:
-        path: |
-          **/node_modules
-        key: ${{ github.run_number }}
-    - name: Lint 
-      run: npm run lint
-
-  build:
-    runs-on: ubuntu-latest
-    needs: install
-    steps:
-    - uses: actions/checkout@v2
-    - uses: actions/setup-node@v1
-      with: 
-        node-version: 14
-        registry-url: 'https://registry.npmjs.org'
-    - name: node_modules cache
-      id: cache
-      uses: actions/cache@v2
-      with:
-        path: |
-          **/node_modules
-        key: ${{ github.run_number }}
-    - name: Build dependencies
-      run: npm run build
+        npm install --legacy-peer-deps --no-audit --no-fund --no-optional
```

Comment:
Maybe adding cache here would be nice ? Maybe using something like this [action](https://github.com/bahmutov/npm-install)

Reply:
I've removed cache as it's not working as expected with workspaces as they use symbolic links for dependencies that are already available in the workspace.